6/6.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+    <head>
+            <link rel="stylesheet" href="6.css" />
+            <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+    </head>
+    <body>
+        <div class="blacksmile">
+                <i class="fas fa-smile"> </i>
+        </div>
+        <div class="ytbutton">
```

Comment:
Use Hyphen Delimited Strings for class names.
Please read just "Use Hyphen Delimited Strings" section from here:
https://medium.freecodecamp.org/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849